api/v1alpha1/common_types.go

```diff
@@ -34,6 +37,14 @@ const (
 	AnnotationsDomain string = "saas.3scale.net"
 )
 
+var (
+	defaultVaultRefreshInterval      metav1.Duration                      = metav1.Duration{Duration: time.Minute}
```

Comment:
This is usually written like this:

```suggestion
	defaultVaultRefreshInterval      metav1.Duration                      = metav1.Duration{Duration: 1 * time.Minute}
```

Reply:
Or `60 * time.Second` as seconds is probably the order of magnitude we would usually work with in this context.

Reply:
```suggestion
	defaultVaultRefreshInterval      metav1.Duration                      = metav1.Duration{Duration: 60 * time.Second}
```